modules/machineset-creating.adoc

```diff
@@ -137,6 +139,8 @@ spec:
       machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone>
```

Comment:
I am so sorry, it should be added above this line.
```
    machine.openshift.io/cluster-api-machine-role: <machine_label> 
    machine.openshift.io/cluster-api-machine-type: <machine_label>
```

Reply:
@sunzhaohua2, no worries! I've relocated the lines.